x-pack/plugins/ml/public/application/license/ml_client_license.ts

```diff
@@ -5,19 +5,33 @@
  * 2.0.
  */
 
+import type { CoreStart } from '@kbn/core/public';
+import { ML_PAGES } from '../../../common/constants/locator';
 import { MlLicense } from '../../../common/license';
 import { showExpiredLicenseWarning } from './expired_warning';
 
 export class MlClientLicense extends MlLicense {
-  fullLicenseResolver() {
+  constructor(private application: CoreStart['application']) {
+    super();
+  }
+
+  private redirectToKibana() {
```

Comment:
I believe these should still return Promise.reject(); as the originals did.
They are used in the page loading resolver, if any of the promises reject at he beginning, the function will [return early](https://github.com/elastic/kibana/blob/759ea55fd27b61244611f823c0e18bceab87f5f0/x-pack/plugins/ml/public/application/routing/use_resolver.ts#L63).

Reply:
updated in https://github.com/elastic/kibana/pull/139469/commits/e8767d4d0f0819a212345da00464c4a008355e00